server/test/DBTablesTest.cc

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2013-2014 Project Hatohol
+ * Copyright (C) 2013-2015 Project Hatohol
```

Comment:
If this change including in this PR, please update copy right years in other headers and sources.